### PR TITLE
update opnsense/pfsense install docs

### PR DESCRIPTION
**Pull request by mmetc**
As discussed - the old pages have been updated but stay in place, they are linked from the plugins and I need a place to store the details.

For OPNSense there is a two click install procedure in the quickstart docs.
For pfSense the quickstart links to the full page, at least for now.

### CODE REVIEW

File: crowdsec-docs/unversioned/getting_started/installation/pfsense.mdx [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
 pagination_next: getting_started/next_steps
 ---
 
-## TODO BY @MMETC
\ No newline at end of file
+# pfSense package
+
+The CrowdSec package for pfSense is not installed from the official repositories, at least not yet.
+
+Please refer to the [detailed documentation](/getting_started/install_pfsense.md) to install or update it from a release archive.
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/getting_started/installation/pfsense.mdx:12]: You must use full url `/docs/next//getting_started/install_pfsense.md`

File: crowdsec-docs/unversioned/getting_started/installation/opnsense.mdx [hunk 2]
```diff
@@ -5,4 +5,28 @@ pagination_prev: getting_started/pre_requisites
...
+ - `os-crowdsec`, the plugin itself
+ - `crowdsec`
+ - `crowdsec-firewall-bouncer`
+
+Do not enable/start the services from the terminal like you would on a standard freebsd system, because the plugin takes care of that.
+
+Refresh the page and go to `Services > CrowdSec > Overview` to verify the running services and installed configurations.
+
+Great, you now have CrowdSec installed on your system. Have a look at the [post installation steps](/getting_started/next_steps.md) to see how to to configure and optimize it,
+or keep reading [the rest of the plugin documentation](/getting_started/install_opnsense.md)
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/getting_started/installation/opnsense.mdx:32]: You must use full url `/docs/next/getting_started/install_opnsense.md`

**Review by LaurenceJJones — APPROVED** [review]

File: crowdsec-docs/docs/getting_started/install_opnsense.md [hunk 3]
```diff
@@ -4,45 +4,53 @@ title: OPNsense
...
 
-Do not enable/start the agent and bouncer services with `sysrc` or `/etc/rc.conf` like you would on a standard freebsd system, because the plugin takes care of that (in a different way).
+You will find some options under `Services > CrowdSec > Settings`. You will see the first three are enabled by default: Log Processor (previously known as IDS), LAPI and Remediation Component (previously known as IPS).
+You can disable them for testing or if you have special requirements.
 
 The parsers, scenarios and all objects from the [CrowdSec Hub](https://hub.crowdsec.net/) are periodically upgraded. The [crowdsecurity/freebsd](https://hub.crowdsec.net/author/crowdsecurity/collections/freebsd) and [crowdsecurity/opnsense](https://hub.crowdsec.net/author/crowdsecurity/collections/opnsense) collections are installed by default.
 
-### Testing the plugin
+Since crowdsec 1.6.3, private IP networks are whitelisted by default as well. This means for example an IP from a LAN or WAN which is on 192.168.x.y won't get blocked by a local decision
+(cummunity blocklists don't contain private IPs).
```
Comment (LaurenceJJones) [inline crowdsec-docs/docs/getting_started/install_opnsense.md:42]: cummunity??

**Review by LaurenceJJones — CHANGES_REQUESTED** [review]
changing approve to changes, as I will review more deeper

**Review by buixor — APPROVED** [review]

**Review by LaurenceJJones — APPROVED** [review]

**Pull request merged**